includes/pages/guest_login.php

```diff
@@ -1,543 +0,0 @@
-                'language'        => $session->get('locale'),
-                'theme'           => config('theme'),
-                'email_human'     => $email_by_human_allowed,
-                'email_messages'  => $email_messages,
-                'email_goody'     => $email_goody,
-                'email_shiftinfo' => $email_shiftinfo,
-                'email_news'      => $email_news,
-                'mobile_show'     => $mobile_show,
-            ]);
-            $settings->user()
-                ->associate($user)
-                ->save();
-
-            $state = new State([]);
-            if (config('autoarrive')) {
-                $state->arrived = true;
-                $state->arrival_date = new Carbon();
-            }
-            $state->user()
-                ->associate($user)
-                ->save();
-
-            if ($session->has('oauth2_connect_provider') && $session->has('oauth2_user_id')) {
-                $oauth = new OAuth([
-                    'provider'      => $session->get('oauth2_connect_provider'),
-                    'identifier'    => $session->get('oauth2_user_id'),
-                    'access_token'  => $session->get('oauth2_access_token'),
-                    'refresh_token' => $session->get('oauth2_refresh_token'),
-                    'expires_at'    => $session->get('oauth2_expires_at'),
-                ]);
-                $oauth->user()
-                    ->associate($user)
-                    ->save();
-
-                $session->remove('oauth2_connect_provider');
-                $session->remove('oauth2_user_id');
-                $session->remove('oauth2_access_token');
-                $session->remove('oauth2_refresh_token');
-                $session->remove('oauth2_expires_at');
-            }
-
-            // Assign user-group and set password
-            $defaultGroup = Group::find(auth()->getDefaultRole());
-            $user->groups()->attach($defaultGroup);
-            if ($enable_password) {
-                auth()->setPassword($user, $request->postData('password'));
-            }
-
-            // Assign angel-types
-            $user_angel_types_info = [];
-            foreach ($selected_angel_types as $selected_angel_type_id) {
-                $angelType = AngelType::findOrFail($selected_angel_type_id);
-                $user->userAngelTypes()->attach($angelType);
-                $user_angel_types_info[] = $angelType->name;
-            }
-
-            // Commit complete user data
-            $db->commit();
-
-            engelsystem_log(
-                'User ' . User_Nick_render($user, true)
-                . ' signed up as: ' . join(', ', $user_angel_types_info)
-            );
-            success(__('Angel registration successful!'));
-
-            // User is already logged in - that means a supporter has registered an angel. Return to register page.
-            if ($authUser) {
-                throw_redirect(page_link_to('register'));
```

Comment:
This seems to be missing in the new version but is a better default in that case

Reply:
Yes, could be a bit strange to land on the sign-up page again.

Reply:
What part of it would be anoying? Its imho more anoying if i get to the / page as it has even less value (as in_: what would i want do do from there?)

Reply:
„Dies scheint in der neuen Version zu fehlen, ist aber in diesem Fall ein besserer Standard“

Reads like the new one is better… So it should be the same behaviour as in guest login?

Reply:
r-ledigt